server

```diff
@@ -91,7 +91,7 @@ echo_msg() {
 run_cmd() {
   CMD=("$@")
   if [ $DRY_RUN -eq 0 ]; then
-    eval "${CMD[@]}"
+   "${CMD[@]}"
```

Comment:
Looks like this commit is only for debug, you should remove it

Reply:
Fixed